subgraph/auxiliary/src/AnchorMapping.ts

```diff
@@ -0,0 +1,17 @@
+import { StateRootAvailable } from '../generated/MosaicFacilitator/Anchor';
+import { AvailableStateRoot } from '../generated/schema';
+
+export default function handleStateRootAvailable(event: StateRootAvailable): void {
+  const entity = new AvailableStateRoot(
+    `${event.transaction.hash.toHex()}-${event.logIndex.toString()}`,
+  );
+
+  entity.blockHeight = event.params._blockNumber;
```

Comment:
This field is different for m0 and m1 

https://github.com/mosaicdao/mosaic-1/blob/fab3b36e36e2d62da021876d4633c25b42a8e1e6/contracts/anchor/StateRootProvider.sol#L30

https://github.com/mosaicdao/mosaic-contracts/blob/7acd863314cffa2dbab70d7e456035f1a8eed08c/contracts/anchor/Anchor.sol#L48

Reply:
It works with updated abi